### ISSUE TITLE
Issue on scroll, and suggested function: Disable scroll snap, but keep snapping on menu/methods

### ISSUE THREAD

**Issue by Florrpan** [lukehaas/Scrollify#189]
I'm building a website where I'd like to disable the normal snapping on scroll, but keeping the snapping on navigation and methods. . I disabled the snapping with $.scrollify.disable() on <= xx screen size. But that also disable all snapping on move methods. It would be good if we could choose to disable only the regular scroll snapping. 
**Reason**:  Having regular snapping on mobile devices is not optimal for the users, because they sometimes scroll too far and starts to snap on sections below when reading the content. In my case, I built a mobile menu with the move methods, and I'd love for them to work even if scroll snapping is disabled.

**Issue:** StandardScrollElements is causing the snapping to glitch.
Basically, when I add standardScrollElements to the elements I want to scroll normal on, the snapping becomes glitchy and it starts snapping and moving where it shouldn't.
This issues becomes very clear when you add several standard scroll elements after each other.

The issue might be that, since the elements are after each other, they overlaps each other and the scroll can't find the proper snapping point.
Example: When I scroll, I get a #sectionName in the url. Sometimes when I scroll past an section, the sectionname is not updated properly, and the jumping starts when I hit the next element. (sometimes the section doesn't get the correct #sectionname)

I hope you understand what I mean. I love this tool!

**Comment by lukehaas** [comment]
Thanks @Florrpan I'll take your first point into consideration when I next update Scrollify.

Regarding the glitches it sounds like you might have some collapsed sections. This can happen if you have floated content. Scrollify needs all sections to be cleared in order for it to calculate their heights correctly.

**Comment by Florrpan** [comment]
That's good Luukehaas! I updated the first post with a dev site for you to check. But I'll look into the float issues, I might not have cleared it all. Thanks!

**Comment by Florrpan** [comment]
Even after clearing floats everywhere, the issue remain.

If I have 3 sections in a row, but have only the first and the 3rd as a normal scroll element, and the middle is snapping as normal, It all works. But when all 3 have normal scroll, snapping goes wonky.

I don't know if i'm doing anything wrong, if I missed a float or something(don't have much floating elements, but some absolute center divs in each section). It just feels wierd. I might try it on a blank site with no additional code.

**Comment by lukehaas** [comment]
I think you might be using `standardScrollElements` wrongly. `standardScrollElements` is for elements within sections such as maps that need to be scrolled independently, not sections themselves.

**Comment by Florrpan** [comment]
You're right, I missed that part. So basically, I can't have a "normal" scroll on sections? Right now the scroll on touch is rather clumsy. It doesn't follow the movement of the finger. Is there a way to scroll normal on phones through sections, just like standardScrollElements do?

**Comment by Florrpan** [comment]
Now when I understand how StandarScrollElements works I adapted the content for the section after that with overflow: auto on the content container.

Here's another suggestion: Having a settings to determine the amount of X scrolling you can do before snapping to the other section. That would make it so that each developer could adapt it more, and remove the "scrolling to far" issue.

**Comment by lukehaas** [comment]
In 1.0.9 I've added an option called `touchScroll` this gives you the option to disable touch based scrolling.
